### ISSUE TITLE
1/2 degree ocean-ice experiments crash

### ISSUE THREAD

**Issue by nikizadehgfdl** [NOAA-GFDL/MOM6#484]
dev/master/2017.04.17
1/2 degree ocean-ice experiments crash with the following traceback:

```
forrtl: severe (174): SIGSEGV, segmentation fault occurred
Image              PC                Routine            Line        Source
fms_MOM6_SIS2_com  0000000001D352D0  Unknown               Unknown  Unknown
fms_MOM6_SIS2_com  0000000000F35535  mom_coriolisadv_m         223  MOM_CoriolisAdv.F90
fms_MOM6_SIS2_com  00000000020DD883  Unknown               Unknown  Unknown
fms_MOM6_SIS2_com  00000000020B8625  Unknown               Unknown  Unknown
fms_MOM6_SIS2_com  00000000020992DF  Unknown               Unknown  Unknown
fms_MOM6_SIS2_com  0000000000F1B29F  mom_coriolisadv_m         221  MOM_CoriolisAdv.F90
fms_MOM6_SIS2_com  0000000000959D99  mom_dynamics_spli         434  MOM_dynamics_split_RK2.F90
fms_MOM6_SIS2_com  000000000083012F  mom_mp_step_mom_          926  MOM.F90
fms_MOM6_SIS2_com  000000000074F323  ocean_model_mod_m         475  ocean_model_MOM.F90
fms_MOM6_SIS2_com  0000000000402572  MAIN__                    977  coupler_main.F90
```

**Comment by nikizadehgfdl** [comment]
This is a simple OMP issue (some OMP shared directives are missing). 
The non-openmp executable works fine. 
But after fixing the simple bug the model comes down after 5 days with:

FATAL from PE  1056: write_energy : Ocean velocity has been truncated too many times.

So, the capability to run single thread with openmp executable is broken. Last I checked this and it was working  was in 2017.04.02 tag. It needs much more time to find the root cause.

**Comment by nikizadehgfdl** [comment]
This has been fixed as of dev/gfdl/2017.10.05
